presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -1194,6 +1197,33 @@ public static long parseDuration(@SqlType("varchar(x)") Slice duration)
         }
     }
 
+    private static long timeAtTimeZone(ConnectorSession session, long timeWithTimeZone, TimeZoneKey timeZoneKey)
```

Comment:
This method was logically placed at the end of the class, but

1. it's no longer at the end after some rebase
2. keeping it at the end generates conflicts

Consider moving to where it's actually used, just below public `timeAtTimeZone` methods.

Also, you can find new home for `valueToSessionTimeZoneOffsetDiff` too. It too is no longer in any logical place.